src/sfr.cpp

```diff
@@ -223,6 +223,7 @@ namespace sfr {
         SFRField<uint32_t> Kd_index = SFRField<uint32_t>(0, 0, (sizeof(constants::acs::Kd_values) / sizeof(constants::acs::Kd_values[0])) - 1, 0x2506);
         SFRField<uint32_t> Kp_index = SFRField<uint32_t>(0, 0, (sizeof(constants::acs::Kp_values) / sizeof(constants::acs::Kp_values[0])) - 1, 0x2507);
         SFRField<uint32_t> c_index = SFRField<uint32_t>(0, 0, (sizeof(constants::acs::c_values) / sizeof(constants::acs::c_values[0])) - 1, 0x2508);
+        SFRField<uint8_t> target_spin_rate = SFRField<uint8_t>(5.0 * constants::acs::spin_resolution, 0x2509);
```

Comment:
Since this is a uint8_t it should go above the uint32_t. The opcodes will all have to be modified so please check the rest of the codebase for "0x" and update Sean and Jonathan.